src/main/resources/templates/animal/about.ftl

```diff
@@ -0,0 +1,1226 @@
+                    js-comp-ver-6.0.5,vc_responsive
+                </div>
+            </div>
+            <div class="content_inner  ">
+                <style type="text/css" data-type="vc_shortcodes-custom-css-5">
+                    .vc_custom_1472631590873 {
+                        background-image: url(https://i.ibb.co/2hdjQnZ/friends-1149841-1920dark.jpg) !important;
+                        background-position: center !important;
+                        background-repeat: no-repeat !important;
+                        background-size: cover !important;
+                    }
+
+                    .vc_custom_1472637221526 {
+                        padding-top: 58px !important;
+                    }
+
+                    .vc_custom_1474273042405 {
+                        padding-top: 52px !important;
+                    }
+
+                    .vc_custom_1491918499778 {
+                        margin-bottom: -79px !important;
+                        padding-top: 72px !important;
+                    }
+
+                    .vc_custom_1481541074234 {
+                        background-image: url(https://bridge82.qodeinteractive.com/wp-content/uploads/2016/08/background.jpg?id=207) !important;
+                        background-position: center !important;
+                        background-repeat: no-repeat !important;
+                        background-size: cover !important;
+                    }
+
+                    .vc_custom_1472639134641 {
+                        padding-top: 102px !important;
+                        padding-bottom: 38px !important;
+                    }
+
+                    .vc_custom_1472634047354 {
+                        padding-top: 30px !important;
+                        padding-bottom: 30px !important;
+                        background-color: #70949d !important;
+                    }
+
+                    .vc_custom_1492610180771 {
+                        padding-top: 20% !important;
+                        padding-bottom: 20% !important;
+                    }
+
+                    .vc_custom_1481124718179 {
+                        margin-top: -12.2% !important;
+                    }
+
+                    .vc_custom_1492524794936 {
+                        padding-right: 0px !important;
+                        padding-left: 0px !important;
+                    }
+
+                    .vc_custom_1492524801071 {
+                        padding-right: 0px !important;
+                        padding-left: 0px !important;
+                    }
+
+                    .vc_custom_1481540011171 {
+                        padding-top: 67px !important;
+                        padding-bottom: 106px !important;
+                    }
+                </style>
+                <script>
+                    var page_scroll_amount_for_sticky = 900;
+
+                </script>
```

Comment:
Take script out as well

Reply:
Moving makes this script unworking

Reply:
Not  moved and deleted. Its speed of scrolling